cpp/src/arrow/compute/kernels/aggregate_basic_internal.h

```diff
@@ -272,8 +272,230 @@ struct MeanKernelInit : public SumLikeInit<KernelClass> {
 };
 
 // ----------------------------------------------------------------------
-// MinMax implementation
+// Last implementation
+template <typename ArrowType, SimdLevel::type SimdLevel, typename Enable = void>
```

Comment:
```suggestion
template <typename ArrowType, typename Enable = void>
```
As far as I can tell `SimdLevel` is completely unused.

Reply:
You are right - This is an side effect of my copying the existing MinMax template.. Will remove